### ISSUE TITLE
error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by bacongobbler** [fermyon/spin#498]
When trying to run Spin 0.1.0 or Spin 0.2.0 from the releases page, I receive an error:

```
><> spin --version
spin: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

Following [the quickstart guide](https://spin.fermyon.dev/quickstart/) on "Linux: Additional Libraries":

```
><> sudo apt-get install build-essential libssl-dev pkg-config
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
build-essential is already the newest version (12.9ubuntu3).
pkg-config is already the newest version (0.29.2-1ubuntu3).
libssl-dev is already the newest version (3.0.2-0ubuntu1.1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

What dependency am I missing on my system?

Kernel version:

```
><> uname -a
Linux broodlord 5.15.0-30-generic #31-Ubuntu SMP Thu May 5 10:00:34 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

OS information:

```
><> cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

**Comment by bacongobbler** [comment]
Workaround: compiling from source works fine.

```
><> git checkout v0.2.0
><> cargo build --release
><> ./target/release/spin --version
spin 0.2.0 (4ff1c75 2022-05-12)
```

**Comment by bacongobbler** [comment]
fixed in #546.
